### PR TITLE
fix: error page

### PR DESCRIPTION
**Pull request by zalnaRs**
#60

### CODE REVIEW

**Review by Axelen123 — CHANGES_REQUESTED** [review]
Thank you for the pull request

File: svelte.config.js [hunk 1]
```diff
@@ -9,12 +9,11 @@ const config = {
 		// adapter-static has vercel detection, but that does not let you set a custom 404 page easily.
 		// Instead, we have to use a wrapper that generates a vercel config if on vercel...
 		adapter: adapter({
-			pages: 'public',
-			fallback: '404.html'
```
[the remark this answers is not in the document]
Comment (zalnaRs) [reply svelte.config.js:13]: Ok I will add it back. I removed it because it didn't work in dev

**Comment by Axelen123** [comment]
It looks good now

**Review by Axelen123 — APPROVED** [review]

**Comment by xafn** [comment]
thank

**Pull request merged**